app.js

```diff
@@ -12,28 +12,34 @@
     ]
 };
 
-
 // Modal elements
```

Comment:
look - this section keeps `modal` elements, but also another elements. I think that names of variables are so descriptive, that each dev will understand what's going on. We could remove this comment :)